src/main/java/edu/cornell/kfs/fp/batch/service/impl/CreateAccountingDocumentValidationServiceImpl.java

```diff
@@ -22,6 +28,11 @@
 public class CreateAccountingDocumentValidationServiceImpl implements CreateAccountingDocumentValidationService {
     private static final Logger LOG = LogManager.getLogger(CreateAccountingDocumentValidationServiceImpl.class);
     
+    private static final Pattern EXCEPTION_MESSAGE_PATTERN = Pattern.compile(
+            "^(?<exceptionClassname>([\\w$]+\\.)+[\\w$]+(Exception|Error|Throwable))((: ?)(?<detailMessage>.+))?$");
```

Comment:
I'm wondering if this pattern string would be better to keep in a property file.  I am not sure where it is best kept.

Reply:
Ok, I'll see if I can externalize this string to one of our config property files.